### ISSUE TITLE
Compatibility for Enhanced Ecommerce

### ISSUE THREAD

**Issue by GiovanniF92** [TechnicalWebAnalytics/dataLayer-shopify#38]
Great integration but datalayer.push and data are only compatible with standard ecommerce.
Could you have an integration with the Enhanced Ecommerce datalayer.push?

Enhanced Ecommerce (UA) https://developers.google.com/tag-manager/enhanced-ecommerce

Enhanced Ecommerce (GA4) https://developers.google.com/tag-manager/ecommerce-ga4

**Comment by adventuretocode** [comment]
@GiovanniF92   have you done the  Enhanced Ecommerce (GA4)  with same dataLayers

**Comment by Brendonwbrown** [comment]
No, it's pretty clear from the documentation that enhanced ecommerce furnishes specific events that are populated with more ecomm-specific data.

I'd love to see an implementation, since this is Shopify we are working with.

**Comment by adventuretocode** [comment]
@Brendonwbrown  i can implement i need some help from the author.
